homework/Program.cs

```diff
@@ -10,6 +8,215 @@ internal class Program
     {
         static void Main(string[] args)
         {
+            /**
+             * Task 1.1
+             * 
+             *      DUODECIMAL "AA" CHECKER
+             *  
+            */
+
+            Console.WriteLine("Enter the number A:");
+            int a = int.Parse(Console.ReadLine());
+
+            Console.WriteLine("Enter the number B:");
+            int b = int.Parse(Console.ReadLine());
+
+            Console.WriteLine("Number(s) that has AA in the DuoDecimal system:");
+
```

Comment:
Better just change places  of a and b if a>b, and call  discoverJustAA(a, b) in one place